src/components/graph/GraphBackground.tsx

```diff
@@ -0,0 +1,41 @@
+/** ****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import * as React from 'react';
+import { observer } from 'mobx-react-lite';
+import '../../styles/graph.scss';
+import SearchWorkspaceStore from '../../stores/workspace/SearchWorkspaceStore';
+import GraphSearch from './search/GraphSearch';
+
+interface GraphBackgroundProps {
+	searchWorkspaceStore: SearchWorkspaceStore;
+}
+
+function GraphBackground({ searchWorkspaceStore }: GraphBackgroundProps) {
+	const rootRef = React.useRef<HTMLDivElement>(null);
+
+	return (
+		<div className='graph' ref={rootRef}>
+			<i className='th2-logo' />
+			<GraphSearch
```

Comment:
`GraphSearch` value resets after switching workspaces. i think it should be inside `GraphWrapper`

Reply:
Lets move all this markup to `GraphWrapper` we dont need this component.
Both SearchWorkspaceStore and WorkspaceStore have onTimestampSelect & onSavedItemSelect methods